shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -467,11 +469,15 @@
           var lastItemInRow = this.items[idx - 1];
           step(lastItemInRow);
         }
-
```

Comment:
Don't remove this blank line please, it makes things hard to read if we squish them together too much :)